### PR TITLE
10790 rsection loading internal forces

### PR DESCRIPTION
**Pull request by david-liska**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] Attached examples
- [ ] Own example attached to Pull Request

**Test Configuration**:
* RFEM / RSTAB / RSECTION version:
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: examplesRSECTION/high_level_functions_demos/load_cases_combinations_internal_forces_demo.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/*********************************************************************************************
+****************************************** Main **********************************************
+*********************************************************************************************/
+
+var t1 = new Date().getTime();
+
+run("../includes/Tools/clearAll.js");
+
+var load_case = new RSectionLoadCase(undefined, "PERMANENT_G", "First load case", false);
+var load_case2 = new RSectionLoadCase(undefined, "PRESTRESS_P", "First load case", false);
```
Comment (jarabroz) [inline examplesRSECTION/high_level_functions_demos/load_cases_combinations_internal_forces_demo.js:16]: This souble be "Second load case"

File: includes/InternalForces/RsectionInternalForces.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        if (typeof member_no !== "undefined") {
+            this.internal_forces.member_no = member_no;
+        }
+        set_comment_and_parameters(internal_forces, comment, params);
+    }
+    else {
+        console.log("Load case no. " + load_case_no + " doesn't exist");
+    }
+}
+
```
Comment (jarabroz) [inline includes/InternalForces/RsectionInternalForces.js:56]: Could be fine to have also prototype for assigment all internal forces

File: includes/Loading/RsectionLoadCombination.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    name,
+    to_solve,
+    comment,
+    params) {
+    this.load_combination = createBaseLoadCombinations(no, name, to_solve, comment, params);
+}
+
+/**
+ * Sets load cases
+ * @param {Array} loadCases     Load cases
```
Comment (jarabroz) [inline includes/Loading/RsectionLoadCombination.js:26]: let's do it same as in RFEM -> [[load_case_no, factor],[]....]

File: includes/Loading/RsectionLoadCase.js [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+        load_case.action_category = actionCategories[action_category];
+    }
+    if (typeof to_solve !== "undefined") {
+        load_case.to_solve = to_solve;
+    }
+    load_case.name = name;
+    set_comment_and_parameters(load_case, comment, params);
+    return load_case;
+}
+
```
Comment (jarabroz) [inline includes/Loading/RsectionLoadCase.js:64]: Add prototypes for GetLoadCase and GetNo - could be useful if I want to make load combination and I have LoadCase as variable

File: includes/Loading/RsectionLoadCombination.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ * @param {Array} factors       Factors, can be undefined
+ */
+RSectionLoadCombination.prototype.SetLoadCases = function (loadCases,
+    factors) {
+    ASSERT(typeof loadCases != "undefined", "At least one load caes must be defined");
```
Comment (jarabroz) [inline includes/Loading/RsectionLoadCombination.js:31]: typo

**Review by jarabroz — CHANGES_REQUESTED** [review]

File: includes/InternalForces/RsectionInternalForces.js [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+/**
+ * Shows list of all available internal forces system types
+ */
+ function get_internal_forces_system_types () {
+    console.log(Object.keys(internal_forces_system_types));
+};
+
+const internal_forces_system_types = {
+    "U_V" : /*load_cases.E_INTERNAL_FORCES_SYSTEM_PRINCIPAL_AXES_U_V*/"u,v",
```
[the remark this answers is not in the document]
Comment (david-liska) [reply includes/InternalForces/RsectionInternalForces.js:155]: There is no value ("u,v", "y,z") - bug?
load_cases.E_INTERNAL_FORCES_SYSTEM_PRINCIPAL_AXES_U_V
load_cases.E_INTERNAL_FORCES_SYSTEM_AXES_X_Y

[the remark this answers is not in the document]
Comment (david-liska) [reply includes/InternalForces/RsectionInternalForces.js:155]: load_cases.E_INTERNAL_FORCES_SYSTEM_PRINCIPAL_AXES_U_V
load_cases.E_INTERNAL_FORCES_SYSTEM_AXES_X_Y
Enums don't return any value,

**Pull request merged**